controllers/imagerepository_controller.go

```diff
@@ -43,7 +42,7 @@ import (
 )
 
 const (
-	InternalRemoteSecretLabelName = "appstudio.redhat.com/internal"
+	InternalSecretLabelName = "appstudio.redhat.com/internal"
```

Comment:
Not sure about UI, but in my opinion it is useful to left any mark that secret created internally by some routine,
so user is aware it may be modified/removed without his consent